airflow/security/kerberos.py

```diff
@@ -46,7 +46,7 @@
 log = logging.getLogger(__name__)
 
 
-def renew_from_kt(principal: str, keytab: str):
+def renew_from_kt(principal: str, keytab: str, exit_on_fail: bool = False):
```

Comment:
This should default to true to maintain previous behaviour.

Reply:
done